MAINTAINERS.md

```diff
@@ -1,5 +1,104 @@
+
+You should understand the [contributing](https://github.com/opiproject/opi/blob/main/CONTRIBUTING.md)
+guidelines if you want to become a maintainer.
+
+## OPI Maintainer Grant/Revocation Policy
+
+An OPI maintainer is a participant in OPI who has the ability to merge pull
+requests directly into main branch on OPI repositories. This ability grants
+the maintainers the ability to affect the progress of the OPI project in a way
+non-maintainers cannot. Being an OPI maintainer represents a significant level
+of trust to the broader project. Maintainer rights are not granted lightly,
+and will be reviewed as necessary to ensure that trust has not been violated.
+
+### Granting Maintainer Rights
+
+Granting maintainer rights will be considered when a candidate has demonstrated
+the following in their interation with the project:
+
+* Contribution of significant new features to OPI.
+  * Submissions are free of obvious defects.
+  * Submissions do not require many revisions.
+* Consistent and meaningful code reviews of other people's pull requests.
+* Helping others in the community via channels such as the mailing list,
+  GitHub. Issues and PRs, and the Slack channels
+* The individual should have plans for sustained contribution to the project.
+
+The process to grant someone the maintainer role is simple:
+
+* An existing maintainer nominates someone by sending a message to the
+  maintainers slack channel.
+* Existing maintainers discuss the idea of giving the candidate maintainer
+  rights.
+* After discussion has ended, existing maintainers will vote on approving the
+  new maintainer.
+* To recieve maintainership, the candidate must receive a majority of yes votes
+  and zero no votes.
+* One final discussion with the candidate happens to ensure the candidate is
+  ready to accept the maintainer role.
+* If successful, the candidate is added to the appropriate GitHub group to
+  gain maintainer access.
+
+### Revoking Maintainer Rights
+
+There are times when a maintainer will be removed in a peaceful way. For
+example, the maintainers may decide that someone should not have merge rights
+due to their lack of git knowledge. In this case, the maintainers can talk
+to this indvidual and work to revoke their maintainer access.
+
+However, there are times when a maintainer may need to be forcefully
+removeed. When a maintainer acts in a way which is either detrimental to the
+OPI project, it raises a situation which needs to be handled quickly to ensure
+the long-term success and stability of the project. It is possible the
+maintainer in question will have their maintainer rights removed. The process
+for this is as follows:
+
+* Discuss with the individual privately about the concerns. Use email for this,
+  sticking to the facts, and try to reach a resolution on the individuals.
+  behavior. Ensure the individual understands their behavior needs to change.
+* Send an email discussing the individuals behavior to the maintainer mailing
```

Comment:
individual's
and do we have a maintainer mailing list?

Reply:
We do not, but we should create one!